classes/LwcSalagaController.cls

```diff
@@ -0,0 +1,136 @@
+public with sharing class LwcSalagaController {
+	public static final String FIRST_NAME_API_NAME = 'FirstName';
+	public static final String LAST_NAME_API_NAME = 'LastName';
+	public static final String LANGUAGE_API_NAME = 'Language';
+
+	public static final String FIRST_NAME_LABEL = 'First Name';
+	public static final String LAST_NAME_LABEL = 'Last Name';
+	public static final String LANGUAGE_LABEL = 'Language';
+
+	public static final String LAST_MODIFIED_BY_LABEL = 'Last Modified By';
+
+	public class UserInfoField {

```

Comment:
why you put wrappers before the main logic? All wrappers should be located in the end